nixos/tests/systemd-boot.nix

```diff
@@ -93,7 +95,7 @@ in
       machine.wait_for_unit("multi-user.target")
 
       machine.succeed("test -e /boot/loader/entries/nixos-generation-1.conf")
-      machine.succeed("grep 'sort-key nixos' /boot/loader/entries/nixos-generation-1.conf")
+      machine.succeed("grep 'sort-key nixor-default' /boot/loader/entries/nixos-generation-1.conf")
```

Comment:
`nixoR-default` — typo?

Reply:
Looks like this is on purpose:

```
[root@bernie:~]# cat /boot/loader/entries/nixos-generation-208.conf
title NixOS
version Generation 208 NixOS Vicuna 24.11.20240725.3d0bda0 (Linux 6.9.10), built on 2024-07-25
linux /EFI/nixos/zdyjg9hmh9xrssl4bk55ws79vr9clh1y-linux-6.9.10-bzImage.efi
initrd     /EFI/nixos/z61f18hhpavf7xqxvgal3gh1qy40bbf7-initrd-linux-6.9.10-initrd.efi
options init=/nix/store/5i8knq7i2dp7q2kacjwr30f047kxvrnz-nixos-system-bernie-24.11.20240725.3d0bda0/init nohibernate root=fstab splash loglevel=4
machine-id 893c3816533c45ac939653b7816f8a18
sort-key nixos
[root@bernie:~]#

[root@bernie:~]# cat /boot/loader/entries/nixos-generation-209+3.conf
title NixOS
version Generation 209 NixOS Vicuna 24.11.20240725.3d0bda0 (Linux 6.9.10), built on 2024-07-25
linux /EFI/nixos/zdyjg9hmh9xrssl4bk55ws79vr9clh1y-linux-6.9.10-bzImage.efi
initrd   /EFI/nixos/z61f18hhpavf7xqxvgal3gh1qy40bbf7-initrd-linux-6.9.10-initrd.efi
options init=/nix/store/bnrbgjbbzb8a4kb27v2ycs9p033ar88i-nixos-system-bernie-24.11.20240725.3d0bda0/init nohibernate root=fstab splash loglevel=4
machine-id 893c3816533c45ac939653b7816f8a18
sort-key nixor-default
```

r < s

Reply:
should be commented then?

Reply:
See https://github.com/NixOS/nixpkgs/commit/10f4b230b7a46f70444d5fe9036e9de490fb5e4d#diff-14341d580318ebe4f2ce22e4fc94c02f6a56229cdc7ae939728628a47b9e6b39R116
I also commented it in the test.